### DISCUSSION TITLE
All List name Not Visible

### DISCUSSION THREAD

**Discussion by ashutoshpw** [community/community#37213] [Other Feature Feedback, Questions, & Ideas]
I have probably created more than 10 lists and it seems all of my lists are not visible when I tap, Add to List from my GitHub Android Mobile app.

**Comment by aioroy** [comment]
Nhgk

**Reply by Fanynul** [reply]
🤳

**Reply by garywheeler69** [reply]
- [ ] ![image](https://user-images.githubusercontent.com/31312650/200126425-c6fc80a3-f218-4a9c-af03-b013aae0f7ac.png)

**Reply by Qazplmqwerty** [reply]
hey y'all nice to see u guys in he@r having fun

**Reply by Sanibello2751** [reply]
Nice

**Reply by ervanipank05** [reply]
@LKN2GO-JSR

**Reply by Sirgusau** [reply]
Thanks

**Reply by kokl407576** [reply]
Seismic Safety Commission
+1 916-263-5506
https://maps.app.goo.gl/5RXLBxvUrNmsRBdu6

**Reply by Sanibello2751** [reply]
Me too

**Reply by kokl407576** [reply]
[]()https://g.co/finance/INR-MMK

**Reply by robi72090** [reply]
31d7554f985819ea90c9e9f7cf43a5696fa64d82

**Reply by Sirgusau** [reply]
Hi

**Reply by Sirgusau** [reply]
Hello

**Comment by Bao345** [comment]
Git Hub perfect better better for use peoples very good app

**Reply by Sirgusau** [reply]
Hello How are you

**Comment by Slava848** [comment]
> I have probably created more than 10 lists and it seems all of my lists are not visible when I tap, Add to List from my GitHub Android Mobile app.

**Reply by Slava848** [reply]
- [ ] []()_@github-staff _

**Reply by Demczuk** [reply]
R

**Reply by Sirgusau** [reply]
Good

**Reply by yasraubi** [reply]
yaseraubi795@gmail.com

**Reply by Arekn0870** [reply]
> Good

**Reply by Sanibello2751** [reply]
Like

**Reply by Sirgusau** [reply]
Hello

**Comment by Slava848** [comment]
> I have probably created more than 10 lists and it seems all of my lists are not visible when I tap, Add to List from my GitHub Android Mobile app.

**Reply by Slava848** [reply]
> > I have probably created more than 10 lists and it seems all of my lists are not visible when I tap, Add to List from my GitHub Android Mobile app.
> 
>     

> - - [ ] []()_**

**_

**Comment by 1096103013** [comment]
Os52

**Reply by Sirgusau** [reply]
Hi

**Comment by amerops92** [comment]
Aaaaa

**Comment by amerops92** [comment]
10

**Comment by 1384963** [comment]
> Git Hub perfect better better for use peoples very good app

**Reply by Arekn0870** [reply]
🫡

**Comment by vittawat6784** [comment]
แปลไทย

**Reply by Sirgusau** [reply]
Welcome

**Reply by Piyang55** [reply]
> แปลไทย

**Reply by Gconquer** [reply]
![002.gif](https://user-images.githubusercontent.com/91039418/221178508-a2ec39ce-274d-49a4-b29d-1b3e18cb475b.gif)

**Comment by Slava848** [comment]
- [ ] - []()

**Reply by Sirgusau** [reply]
Hi

**Comment by hsghn** [comment]
Yes

**Reply by hsghn** [reply]
Reply

**Reply by hsghn** [reply]
Reply

**Reply by Sirgusau** [reply]
Hi

**Reply by ultrabos1429** [reply]
Go

**Comment by jasti99** [comment]
https://github.com/raghavkhanna30/Telegram-Scraper/blob/master/smsbot.py

**Reply by Qazplmqwerty** [reply]
Whats this ???

**Reply by ultrabos1429** [reply]
@LKN2GO-JSR

**Comment by cmtfelix** [comment]
L

**Comment by Damhafi** [comment]
quem é voces


Alireza ***@***.***> escreveu no dia sexta, 30/12/2022 à(s)
14:42:

**Comment by arfanakash446** [comment]
> I have probably created more than 10 lists and it seems all of my lists are not visible when I tap, Add to List from my GitHub Android Mobile app.

**Comment by Shahzamin345** [comment]
0599486240

**Comment by Nwode** [comment]
Happy new year

**Reply by Sirgusau** [reply]
Thanks

**Reply by ultrabos1429** [reply]
- [x] Thanks

**Reply by Arekn0870** [reply]
🫡

**Comment by Tahamakki1234** [comment]
Taha

**Comment by kokl407576** [comment]
2https:/youtube.com/@europeanparliament

**Comment by Sal667** [comment]
@vittawat6784

**Reply by Sal667** [reply]
Leave replya

**Reply by Sal667** [reply]
> @vittawat6784 

Auguri ❤

**Reply by Sirgusau** [reply]
Hello How are you

**Reply by Sirgusau** [reply]
Same here

**Comment by ivanovev9enii278** [comment]
- [ ] ```


```

**Reply by ANGELAFN75** [reply]
Have

**Reply by robi72090** [reply]
0xE75c21D99C805B048d4622216317F563AD519430

**Comment by ultrabos1429** [comment]
# - uae 
# - USB 
# - EA
- [x] ID
# EVERYTHING 
# Quite 
# Email 
# Number of people 
# Number of code 426655

**Reply by drefcot17th** [reply]
Goog

**Reply by drefcot17th** [reply]
F

**Comment by Octocat5B** [comment]
I want to try this but everytime I use my lobster, MoneyGram doesn't works any help?

**Comment by Maera-Dennis** [comment]
Good

**Comment by jamshidkawa** [comment]
Hi

**Reply by Medusa57** [reply]
Hi

**Reply by Qazplmqwerty** [reply]
Hi

**Reply by Sirgusau** [reply]
Hrllo

**Reply by Sirgusau** [reply]
Hello

**Reply by Sirgusau** [reply]
Hello

**Reply by Qazplmqwerty** [reply]
I can see your owners have been bad influence on u I hope u move toward the light and away from this darkness ass u get control over your own life

**Comment by Qazplmqwerty** [comment]
Hi


bigudabest

**Reply by Qazplmqwerty** [reply]
Hello I just took control of this again my phyco ex friend and her son have been using it in my name

**Comment by Sanibello2751** [comment]
Nil

**Comment by Hshihbi** [comment]
Github  is very talented

**Reply by Hshihbi** [reply]
Yes

**Reply by Hshihbi** [reply]
How about  you?

**Comment by aktivasi123** [comment]
![IMG-20230112-WA0001.jpg](https://user-images.githubusercontent.com/119670742/220871640-1c91a71e-f666-4d7b-a71a-9c917c17272e.jpg)

**Reply by aktivasi123** [reply]
Siaran Pers: OJK Peringatkan Kresna Life Terkait Persetujuan Tertulis Konversi Kewajiban Kepada Pemegang Polis Menjadi Subordinate Loan (SOL)

16 FEBRUARI 2023

​￼SP - OJK PERINGATKAN KRESNA LIFE TERKAIT PERSETUJUAN TERTULIS KONVERSI KEWAJIBAN KEPADA PEMEGANG POLIS MENJADI SUBORDINATE LOAN (SOL).PDF

 

​SP 20/GKPB/OJK/II/2023

 

SIARAN PERS

OJK PERINGATKAN KRESNA LIFE TERKAIT PERSETUJUAN TERTULIS KONVERSI KEWAJIBAN KEPADA PEMEGANG POLIS MENJADI SUBORDINATE LOAN (SOL)

Jakarta, 16 Februari 2023. Kepala Eksekutif Pengawas Perasuransian Dana Pensiun Lembaga Pembiayaan dan Lembaga Jasa Keuangan Lainnya Ogi Prastomiyono menyampaikan OJK sampai saat ini belum menerima dari Kresna Life dokumen pernyataan tertulis dari setiap pemegang polis terkait persetujuan atas skema konversi kewajiban menjadi pinjaman subordinasi (SOL). Dokumen tersebut telah diminta untuk disampaikan paling lambat pada 13 Februari 2023.

Ogi menjelaskan, dalam RPK terakhir yang disampaikan Kresna Life, tidak ada alternatif tambahan setoran modal dari PSP atau menggandeng strategic investor, tetapi dengan skema konversi kewajiban kepada pemegang polis menjadi SOL. Untuk itu, diperlukan adanya persetujuan tertulis dari pemegang polis setelah mereka diberikan pemahaman yang komprehensif mengenai SOL termasuk konsekuensinya. Selain itu, apabila jumlah konversi SOL belum cukup untuk perhitungan rasio solvabilitas, maka PSP harus menyetorkan tambahan modal sampai dengan rasio solvabilitas terpenuhi.

Perubahan kewajiban kepada pemegang polis menjadi SOL akan menyehatkan keuangan perusahaan, namun tidak dapat membantu likuiditas karena tidak ada aliran dana masuk sebagai tambahan permodalan.

Sehubungan dengan konversi dimaksud terdapat beberapa poin terkait dengan konversi SOL yang perlu mendapat perhatian publik, di antaranya:

Konversi SOL wajib mendapatkan persetujuan dari pemegang polis secara tertulis, sehingga Kresna Life harus menyampaikan dokumen yang ditandatangani oleh pemegang polis yang menyatakan kesediaannya untuk konversi SOL. 

Sampai dengan batas waktu yang ditentukan, OJK belum menerima dokumen persetujuan tertulis dari setiap pemegang polis terkait konversi SOL. Persetujuan tertulis dibutuhkan untuk perhitungan solvabilitas Perusahaan. Apabila berdasarkan perhitungan masih terdapat kekurangan, maka Pemegang Saham Pengendali harus menambah modal untuk menutupi kekurangan tersebut dan dituangkan dalam RPK Kresna Life.

Untuk dapat diperhitungkan dalam solvabilitas, skema SOL mempunyai beberapa kriteria yang memiliki potensi risiko bagi pemegang polis sehingga OJK meminta Kresna Life menyampaikan risiko tersebut secara transparan kepada pemegang polis. Risiko itu antara lain: 

a. Kedudukan pemegang polis sebagai pemberi pinjaman subordinasi (SOL) secara otomatis melepaskan haknya atas pembagian dana jaminan Kresna Life. 

b. Pemberi pinjaman SOL tidak dapat mencairkan dananya apabila Kresna Life belum dapat memenuhi rasio tingkat kesehatan. 

c. Terdapat ketidakpastian pengembalian pinjaman SOL karena tidak ada tambahan uang masuk ke Kresna Life sehingga sangat tergantung pada kinerja Kresna Life. 

d. Pinjaman subordinasi hanya dapat memberikan tingkat bunga paling tinggi 1/5 dari tingkat bunga Bank Indonesia.

e. Pemberi Pinjaman Subordinasi memiliki prioritas pembayaran lebih rendah jika dibandingkan dengan pembayaran kewajiban kepada pemegang polis dalam urutan pembagian aset jika Kresna Life dilikuidasi.

Untuk menangani defisit Kresna Life, OJK juga telah meminta pertanggungjawaban pemegang saham pengendali untuk menambah modal. Namun hingga saat ini, Penambahan modal tersebut belum direalisasikan.

Pada tanggal 31 Januari 2020 terdapat akumulasi dana masuk dari PT Duta Makmur Sejahtera sebagai salah satu pemegang saham Kresna Life sebesar Rp325 miliar, tetapi pada hari yang sama hampir seluruh dana tersebut berpindah kepada Perusahaan afiliasi grup Kresna. Kresna Life tidak melaporkan masuknya dana sebesar Rp325 miliar tersebut kepada OJK sebagai setoran modal, sehingga secara ketentuan tidak dapat diakui sebagai tambahan modal.

Perusahaan asuransi yang dikenakan sanksi PKU (Pembatasan Kegiatan Usaha) tetap wajib melakukan pembayaran klaim saat ada klaim yang jatuh tempo. Begitu juga Kresna Life, harus membayar setiap klaim yang telah jatuh tempo. Apabila hal tersebut tidak dilakukan, maka perusahaan dapat dikategorikan gagal bayar.

Apabila Perusahaan tidak dapat menyampaikan dokumen persetujuan tertulis dari setiap pemegang polis terkait konversi SOL dan/atau rencana penambahan modal untuk menutupi kekurangan solvabilitas yang dituangkan dalam RPK Kresna Life, OJK akan memberikan tindakan tegas karena kesempatan perbaikan RPK sudah diberikan waktu yang cukup.

 

***


Informasi lebih lanjut:


Kepala Grup Komunikasi Publik OJK Darmansyah

Telp. (021) 29600000; Email: humas@ojk.go.id

Artikel Lain

**Comment by abdolahnezhad** [comment]
please help  me :because i have 34598921.74 pvt-554(bep2network)
in my account trust wallet and l don't know, how l to convert these tokens  to new asset or convert to BNB?
thank you sincerely  Mohamad

**Reply by abdolahnezhad** [reply]
hi team GitHub. 
i don't understand to using github for. how i  to convert pvt-554 tokens  with BNB coin?? ..please guide me

**Reply by abdolahnezhad** [reply]
please help  me :because i have 34598921.74 pvt-554(bep2network)
in my account trust wallet and l don't know, how l to convert these tokens  to new asset or convert to BNB?
thank you sincerely  Mohamad

**Reply by Sweetilona3** [reply]
[

**Comment by Sirgusau** [comment]
Hello Mohammed
I received your mail about your questions but kindly expanciate the
Blockchain application you are using together with the wallet you want to
use for the transfer.
Thanks

**Reply by Qazplmqwerty** [reply]
Sick Pedro stalkers the person using this devise is my ex its her devise but its supposed to be mine its her way of being a sicko and hiding as me please stop helping her with her deamonds u all need help

**Comment by abdolahnezhad** [comment]
thank you for your email..
i have 34598921 pvt-554 (bep2network) in trust wallet.
i decided these tokens (pvt-554)to convert BNB coin.but i can't. please
guide me.
sincerely Mohamad

**Comment by abdolahnezhad** [comment]
thank you for your email..
i have 34598921 pvt-554 (bep2network) in trust wallet.
i decided these tokens (pvt-554)to convert BNB coin.but i can't. please
guide me.
sincerely Mohamad

**Reply by emmymxny** [reply]
Hi

**Comment by abdolahnezhad** [comment]
hi admin
i agree to pay 2pvt to you..please guide me

**Comment by abdolahnezhad** [comment]
It works!

**Comment by Mehdimome** [comment]
Very good

**Comment by Qazplmqwerty** [comment]
U all are sick stalker pedos and need help

bigudabest

**Comment by murilloggomes** [comment]
me ajudando em um scsn

**Comment by abodaa198911** [comment]
عاعه

**Comment by abodaa198911** [comment]
abodaa1989111m@gmail.com

**Comment by George-cb** [comment]
Make Github a lot easier to use for beginners and show clear information and  instructions, Please!!

**Comment by George-cb** [comment]
How are you going to do this. You have to be ready to do this Now!!

**Comment by George-cb** [comment]
Make My Lists Visible to others.

**Comment by George-cb** [comment]
Help me out with using Github,please!

**Comment by Weirdo5000** [comment]
loveisweird85@gmail.com

**Comment by takenbythebest** [comment]
how do I get my phone normal without any problems I have the revvel6 5g

**Reply by 0583231610** [reply]
> how do I get my phone normal without any problems I have the revvel6 5g
> 

.

**Reply by BezRegi** [reply]
reply

**Reply by BezRegi** [reply]
> how do I get my phone normal without any problems I have the revvel6 5g
>

**Reply by mehdimp72947294** [reply]
> > how do I get my phone normal without any problems I have the revvel6 5g
> > 
> 
> .

**Comment by Mewjaden** [comment]
Approved

**Reply by BezRegi** [reply]
> Approved

**Comment by 37elivan73** [comment]
Hola gn que es una cartera o bank

**Reply by 37elivan73** [reply]
No es hack es solo un pregunta

**Reply by 37elivan73** [reply]
> Someone Hack me from over there

It's not hack it's just a question y that's all

**Reply by 37elivan73** [reply]
Yo soy de México

**Reply by 37elivan73** [reply]
I only asked you a question

**Reply by 37elivan73** [reply]
I only asked you a question

**Comment by 37elivan73** [comment]
> - [ ] ```
> 
> 
> ```

show me and catch up

**Comment by dapurliwet** [comment]
My name warawiri_3526 my hobby is singing, the unique is in me singing songs with Good Preparation and singing songs that were previously unknown, had not been sung only based on feeling, apprecistion, imagination and improvisation. I Can Sing 7000 song in months. Thank you very much my friend 👌🙏

**Comment by Minhanhac** [comment]
> Git Hub perfect better better for use peoples very good app
]()

**Reply by abdolahnezhad** [reply]
Changed paths:A new asset           please help  me :because i have 34598921.74 pvt-554(bep2network)
in my account trust wallet and l don't know, how l to convert these tokens  to new asset or convert to BNB?
thank you sincerely  Mohamad

**Comment by Sweetilona3** [comment]
> I have probably created more than 10 lists and it seems all of my lists are not visible when I tap, Add to List from my GitHub Android Mobile app.

**Reply by tammynav12** [reply]
So what bitch

**Comment by tammynav12** [comment]
> Your Google verification code is 166033
>

**Comment by 37elivan73** [comment]
Bueno que te traes hijo de tu reputa madreEl 6 may. 2023 4:18 p. m., Enrique Navarrete ***@***.***> escribió:
Bitch

**Comment by 37elivan73** [comment]
Basta de llamarme así pinche monito cilindreroEl 6 may. 2023 4:18 p. m., Enrique Navarrete ***@***.***> escribió:
Bitch

**Comment by tammynav12** [comment]
You are a bitch you dumb ass


Sent from Yahoo for iPhone


On Sunday, May 7, 2023, 1:16 AM, 37elivan73 ***@***.***> wrote:



Basta de llamarme así pinche monito cilindreroEl 6 may. 2023 4:18 p. m., Enrique Navarrete ***@***.***> escribió:
Bitch

**Reply by Acinen** [reply]
Ok

**Comment by tammynav12** [comment]
Fuck you whore


Sent from Yahoo for iPhone


On Sunday, May 7, 2023, 1:15 AM, 37elivan73 ***@***.***> wrote:



Bueno que te traes hijo de tu reputa madreEl 6 may. 2023 4:18 p. m., Enrique Navarrete ***@***.***> escribió:
Bitch

**Comment by Minhanhac** [comment]
Vào CN, 7 thg 5, 2023 lúc 16:49 Enrique Navarrete ***@***.***>
đã viết:

> Fuck you whore
>
>
> Sent from Yahoo for iPhone
>
>

**Comment by 37elivan73** [comment]
El 6 may. 2023 4:18 p. m., Enrique Navarrete ***@***.***> escribió:
Bitch

**Comment by 37elivan73** [comment]
Tengo que deja que me insulten para que todo esté bien no lo creo como 7correos llamándome "bitch" en un principio la ignore pero logro enfurecer me por las demás personas siento hallan tenido que leer este tipo de conducta.El 6 may. 2023 11:14 p. m., ivan galicia ***@***.***> escribió:Bueno que te traes hijo de tu reputa madreEl 6 may. 2023 4:18 p. m., Enrique Navarrete ***@***.***> escribió:
Bitch

**Comment by 37elivan73** [comment]
El 7 may. 2023 2:46 a. m., ivan galicia ***@***.***> escribió:Tengo que deja que me insulten para que todo esté bien no lo creo como 7correos llamándome "bitch" en un principio la ignore pero logro enfurecer me por las demás personas siento hallan tenido que leer este tipo de conducta.El 6 may. 2023 11:14 p. m., ivan galicia ***@***.***> escribió:Bueno que te traes hijo de tu reputa madreEl 6 may. 2023 4:18 p. m., Enrique Navarrete ***@***.***> escribió:
Bitch

**Comment by tammynav12** [comment]
Enrique Navarrete who just beat his wife andvleft her with nothing he was cheating on her with 3 females from this community thing he needs to block or the females what he did to my sister was not right 


Sent from Yahoo for iPhone


On Sunday, May 7, 2023, 3:47 AM, tammy oreilly ***@***.***> wrote:

You are a bitch you dumb ass


Sent from Yahoo for iPhone


On Sunday, May 7, 2023, 1:16 AM, 37elivan73 ***@***.***> wrote:



Basta de llamarme así pinche monito cilindreroEl 6 may. 2023 4:18 p. m., Enrique Navarrete ***@***.***> escribió:
Bitch

**Comment by tammynav12** [comment]
Sent from Yahoo for iPhone


On Sunday, May 7, 2023, 6:03 AM, tammy oreilly ***@***.***> wrote:

Enrique Navarrete who just beat his wife andvleft her with nothing he was cheating on her with 3 females from this community thing he needs to block or the females what he did to my sister was not right 


Sent from Yahoo for iPhone


On Sunday, May 7, 2023, 3:47 AM, tammy oreilly ***@***.***> wrote:

You are a bitch you dumb ass


Sent from Yahoo for iPhone


On Sunday, May 7, 2023, 1:16 AM, 37elivan73 ***@***.***> wrote:



Basta de llamarme así pinche monito cilindreroEl 6 may. 2023 4:18 p. m., Enrique Navarrete ***@***.***> escribió:
Bitch

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>



I’m—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you commented.Message ID: ***@***.***>
It’s this horrible nasty who has sexually problem with women he needs help

**Comment by tammynav12** [comment]
You are the bitch who cheats on his then beats her leaves her and uses all her email. I say ban this guy


Sent from Yahoo for iPhone


On Sunday, May 7, 2023, 3:47 AM, tammy oreilly ***@***.***> wrote:

You are a bitch you dumb ass


Sent from Yahoo for iPhone


On Sunday, May 7, 2023, 1:16 AM, 37elivan73 ***@***.***> wrote:



Basta de llamarme así pinche monito cilindreroEl 6 may. 2023 4:18 p. m., Enrique Navarrete ***@***.***> escribió:
Bitch

**Comment by tammynav12** [comment]
Yes 


Sent from Yahoo for iPhone


On Sunday, May 7, 2023, 6:09 AM, tammy oreilly ***@***.***> wrote:

You are the bitch who cheats on his then beats her leaves her and uses all her email. I say ban this guy


Sent from Yahoo for iPhone


On Sunday, May 7, 2023, 3:47 AM, tammy oreilly ***@***.***> wrote:

You are a bitch you dumb ass


Sent from Yahoo for iPhone


On Sunday, May 7, 2023, 1:16 AM, 37elivan73 ***@***.***> wrote:



Basta de llamarme así pinche monito cilindreroEl 6 may. 2023 4:18 p. m., Enrique Navarrete ***@***.***> escribió:
Bitch

**Reply by tammynav12** [reply]
> Yes  Sent from Yahoo for iPhone On Sunday, May 7, 2023, 6:09 AM, tammy oreilly ***@***.***> wrote: You are the bitch who cheats on his then beats her leaves her and uses all her email. I say ban this guy Sent from Yahoo for iPhone On Sunday, May 7, 2023, 3:47 AM, tammy oreilly ***@***.***> wrote: You are a bitch you dumb ass Sent from Yahoo for iPhone On Sunday, May 7, 2023, 1:16 AM, 37elivan73 ***@***.***> wrote: Basta de llamarme así pinche monito cilindreroEl 6 may. 2023 4:18 p. m., Enrique Navarrete ***@***.***> escribió: Bitch —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***> — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you commented.Message ID: ***@***.***>

I want Mr. Enrique Navarrete to stop using my name any name with tammynav is my name it his

**Comment by tammynav12** [comment]
tammynav12 is using my information for his account his name is Enrique Navarrete and his phone number is 708-833-6218 please remove my email sir!!

**Comment by 734485427** [comment]
> I have probably created more than 10 lists and it seems all of my lists are not visible when I tap, Add to List from my GitHub Android Mobile app.

**Comment by Lonelydark516** [comment]
Can I work with you ?

**Reply by ttdollar** [reply]
Yes I will be glad

**Reply by Sanibello2751** [reply]
Yes

**Reply by Mewjaden** [reply]
What's needed to be done ?

**Comment by Mewjaden** [comment]
Need my approval @ashutoshpw
